training/general/src/environments/acpt-pytorch-1.11-py38-cuda11.3-gpu/context/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu113-py38-torch1110:{{latest-image-tag:monthly\.\d{6}}}
+
+# Install pip dependencies
+RUN pip install 'ipykernel~=6.0' \
+		'azureml-core=={{latest-pypi-version}}' \
+		'azureml-dataset-runtime=={{latest-pypi-version}}' \
+		'azureml-defaults=={{latest-pypi-version}}' \
+		'azure-ml=={{latest-pypi-version}}' \
+		'azure-ml-component=={{latest-pypi-version}}' \
+		'azureml-mlflow=={{latest-pypi-version}}' \
+		'azureml-telemetry=={{latest-pypi-version}}' \
+                'torch-tb-profiler~=0.4.0' \
+				'py-spy==0.3.12'
```

Comment:
shouldn't the py-spy be aligned with the rest?

Reply:
you mean different version? I don't see py-spy added to other envs.